### ISSUE TITLE
Update scripts with lib paths

### ISSUE THREAD

**Issue by frankdoom** [ptarmiganlabs/butler#23]
:) sense scripts included point to http:// but should rather use lib paths or variables to be defined in Sense script

**Comment by mountaindude** [comment]
You are absolutely right - but there is a catch.
The reason for the http is that back in the day (and maybe even still in current Sense versions, I should check) it was possible to modify the URL of a web data connection if it was defined using http... (which is what Butler for example uses to send Slack messages), but not if it was defined using lib://. 

For example, when sending a Slack message, the helper function that sends Slack messages will behind the scenes encode your message, then build a URL (pointing to the Butler server) that is finally called. I.e. we need a data connection where the URL can be modified in runtime (when the script is executing). 

I need to check, but I think the "legacy mode" flag still controls whether http://... can be used in the script in web connections. 

Bottom line: This should be looked into. Maybe we can use a REST connector instead... 0r change the architecture all together. Butler was originally created for Sense 2.x - lots have changed (to the better with more possibilities and options) since then.
